hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/converter/TestFSQueueConverter.java

```diff
@@ -508,29 +521,29 @@ public void testReservationSystemNotSupported() {
   }
 
   private void assertNoValueForQueues(Set<String> queues, String postfix,
-      Configuration config) {
+      CapacitySchedulerConfiguration capacitySchedulerConfig) {
     for (String queue : queues) {
       String key = PREFIX + queue + postfix;
       assertNull("Key " + key + " has value, but it should be null",
-          config.get(key));
+          capacitySchedulerConfig.get(key));
```

Comment:
I think we should also check where we are calling this method, because in some cases the simple get should be changed to the corresponding convenience method here as well.